web-lib-funcs.pl

```diff
@@ -969,6 +969,11 @@ sub PrintHeader
 {
 my ($cs, $mt) = @_;
 $mt ||= "text/html";
+my %miniserv;
+&get_miniserv_config(\%miniserv);
+if (!$gconfig{'no_strict_transport_security'} && $miniserv{'ssl'}) {
```

Comment:
Instead of reading the miniserv conf, just check if `$ENV{'HTTPS'}` equals `'ON'`

Reply:
Alright! Fixed.